Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "dprint-plugin-exec"
+version = "0.0.3"
```

Comment:
We already did a couple of releases so I set it higher than in private repo. However, I don't mind to set it to `0.0.1`